rpm/libvncserver.spec

```diff
@@ -2,7 +2,7 @@
 Name:           LibVNCServer
 Version:        0.9.10
 Release:        1
-License:        GPL
+License:        GPLv2
 Group:          Libraries/Network
```

Comment:
This is actually licensed under multiple licenses, I could find at least:

GPLv2+ and MIT and BSD-2-Clause

Also URL pointing to this git tree would be good to add.